map/src/data/index.ts

```diff
@@ -45,3 +47,23 @@ export const SERVICE_TYPES = Object.keys(SERVICES) as Service[];
 export interface Filter {
   service?: Service;
 }
+
+export const LANGUAGES = {
+  en: {
+    label: t(s => s.languages.en),
+  },
+  ru: {
+    label: t(s => s.languages.ru),
+  },
+};
+
+export type Language = keyof typeof LANGUAGES;
+
+export const isLanguage = (language: string): language is Language =>
+  language in LANGUAGES;
+
+export const LANGUAGE_TYPES = Object.keys(LANGUAGES) as Language[];
+
+export interface Translate {
+  language?: Language;
+}
```

Comment:
This information feels like it's mostly duplicated from `i18n/index.ts`, and would better live there. The `data` directory is more for data we're displaying, not UI implementation (which `i18n` would come under).